maths-club-pack/content/facilitator/coconut-trader.md

```diff
@@ -0,0 +1,42 @@
+# Coconut Trader
+
+## Introduction  
+
+The key to this question is to realise that you should take all the coconuts from one sack until that sack is empty. Then do the same with the second sack. If each time you reach a checkpoint you take one coconut from each sack, then you would have no coconuts left by the time you reach the market! Leave students to work this out for themselves but if they get stuck then give them a hint.
+
+
+
+## Solution  
+
+<ins>Emptying the first sack<ins>
+
+You currently have three sacks so at each checkpoint you must give three coconuts each time. That means that the sack will be empty after 30 ÷ 3 = 10 checkpoints.You throw away this empty sack.  
```

Comment:
I think the author wanted to emphasise the word **three** in this line, so please can you make it bold. Similarly the words **two** and **one** should be bold in the other solution parts
![image](https://user-images.githubusercontent.com/10515065/91466313-cb54b300-e843-11ea-8d17-3577408571a2.png)